response_operations_ui/templates/collection_exercise/ce-collection-instrument-eq-section.html

```diff
@@ -18,27 +18,75 @@
                 })
             }}
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-            <input type="hidden" name="ce_id" value="{{ ce.id }}" /> 
-            {% set selectAllText = "Select all" %}
-            {% set unselectAllText = "Unselect all" %}
-            {% if collection_instruments['EQ']|length == all_cis_for_survey|length %}
-                {% set autoSelectState = unselectAllText %}
-            {% else %}
-                {% set autoSelectState = selectAllText %}
-            {% endif %}
-            {{ 
-                onsButton({
-                    "html": '<span class="ons-js-button-text" id="selection-text">' + autoSelectState + '</span>',
-                    "variants": btnClasses,
-                    "classes": 'ons-u-mb-m ons-btn--small ons-btn--secondary',
-                    "id": "select-or-unselect-ci",
-                    "attributes": {
-                        "data-unselect-all": unselectAllText,
-                        "data-select-all": selectAllText
-                    }
-                })
-            }}
-            <div class="ons-grid ons-col-12@m ons-grid-flex--vertical-center ons-grid--no-wrap@m ons-u-mt-m ons-u-mb-m">
+            <input type="hidden" name="ce_id" value="{{ ce.id }}" />
+              {% if cir_enabled %}
+                {% set ciSubmitButtonText = "Continue to choose versions" %}
+                <div class="ons-grid ons-col-12@m ons-grid-flex--vertical-center ons-grid--no-wrap@m ons-u-mt-m ons-u-mb-m">
+                  <div class="ons-grid__col ons-col-4@m" style="padding-left: 1rem">
+                    <table id="collection-instruments-left-table" class="ons-table ons-table--responsive">
+                      <thead class="ons-table__head">
+                        <tr class="ons-table__row" style="white-space:nowrap;">
+                          <th scope="col" class="ons-table__header">
+                            <span class="ons-table__header-text">EQ formtype</span>
+                          </th>
+                          <th scope="col" class="ons-table__header">
+                            <span class="ons-table__header-text">CIR version</span>
+                          </th>
+                        </tr>
+                      </thead>
```

Comment:
I cant't remember exactly how this works, but the table is quite a bit slimmer than the design when I deploy it
<img width="1437" alt="image" src="https://github.com/user-attachments/assets/39d2a9dc-61a0-47f1-9b58-3cbb6d556dc5" />

Reply:
I presumed that it will will expand once the data is supplied. However, I will check if this needs to be a default width.

Reply:
I have got confirmation from Karen and Alina, this is fine as it is. They will add a story if this changes.